contracts/contracts/rem.attr/src/rem.attr.cpp

```diff
@@ -0,0 +1,115 @@
+#include <rem.attr/rem.attr.hpp>
+
+
+namespace eosio {
+
+   void attribute::confirm( const name& owner, const name& attribute_name )
```

Comment:
@me-vadim as we do `attr.pending.clear();` should we check for `nothing to confirm` ?

Reply:
Done